rcldotnet_tests/MessagesTest.cs

```diff
@@ -38,9 +38,7 @@ public void SetInt64Data()
         public void SetStringData()
         {
             std_msgs.msg.String msg = new std_msgs.msg.String();
-            //NOTE(sam): msg.Data now initializes to null and not <string.Empty>, intentional?
-            // Assert.That(msg.Data, Is.EqualTo(""));
-            Assert.That(msg.Data, Is.EqualTo(null));
+            Assert.That(msg.Data, Is.EqualTo(""));
```

Comment:
Yeah I fixed that. Strings need to be initialized in order for Marshalling to succeed :]